runtime/parsing.c

```diff
@@ -277,7 +281,7 @@ CAMLprim value caml_parse_engine(struct parser_tables *tables,
                                 /* The ML code calls the semantic action */
   case SEMANTIC_ACTION_COMPUTED:
     RESTORE;
-    Field(env->s_stack, sp) = Val_int(state);
+    Store_field(env->s_stack, sp, Val_int(state));
```

Comment:
Revert this change, and others in this file. `Field` is an l-value.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/162a25c57a605107da49ec258a10ce14e8302759